controllers/configurationpolicy_controller.go

```diff
@@ -2911,7 +2908,7 @@ func convertPolicyStatusToString(plc *policyv1.ConfigurationPolicy) (results str
 			}
 		}
 	}
-
+	policyLog.Info("Policy status message", "status", result)
```

Comment:
Having the log inside this function might result in it being duplicated sometimes, but I think that is a better problem than missing it sometimes. I wonder if it should be behind `V(1)` or a special flag, to keep things from getting too noisy?

Reply:
The goal is to have this logged by default so servicing systems that only have the GRC governance metrics and logs can determine the NonCompliance messages.  Since the NonCompliance message isn't in a metric we need it to always be in log.  I'll have to see if the V(1) logs show up by default.  I'll do a duplication detection too to see if that happens -- it looks like I've cleaned up the logs I captured last week so will recollect.

Reply:
I agree with @JustinKuli. I think these could be very noisy and make the logs grow quickly--I'd prefer we ask that servicing systems bump the log level if they want the information present in the logs. It's not too difficult since we have a custom annotation that can be applied to the ManagedClusterAddon. Does SD have a requirement that the status message be present at default logging levels?